core/server/master/src/main/java/alluxio/master/file/meta/InodeTree.java

```diff
@@ -219,6 +215,11 @@ public InodeTree(InodeStore inodeStore, ContainerIdGenerable containerIdGenerato
     mMountTable = mountTable;
   }
 
+  @Override
+  public String getName() {
+    return mState.getName();
```

Comment:
Does this cause a problem later if we need to add something other than `mState` (as unlikely as that is)